MobileCenter/MobileCenter/Internals/Storage/MSFileUtil.m

```diff
@@ -97,15 +97,22 @@ + (nullable NSData *)dataForFile:(MSFile *)file {
   NSFileManager *fileManager = [NSFileManager defaultManager];
 
   // Check validity.
-  if (!directoryPath || !fileExtension || ![fileManager fileExistsAtPath:directoryPath]) {
+  if (!directoryPath || !fileExtension) {
+    return nil;
+  }
+
```

Comment:
You can remove this new line. Basically our convention doesn't allow new lines within code blocks if it's not preceded by a comment.